pkg/catalog/egress.go

```diff
@@ -35,7 +35,7 @@ func (mc *MeshCatalog) GetEgressTrafficPolicy(serviceIdentity identity.ServiceId
 	var trafficMatches []*trafficpolicy.TrafficMatch
 	var clusterConfigs []*trafficpolicy.EgressClusterConfig
 	portToRouteConfigMap := make(map[int][]*trafficpolicy.EgressHTTPRouteConfig)
-	egressResources := mc.policyController.ListEgressPoliciesForSourceIdentity(serviceIdentity.ToK8sServiceAccount())
+	egressResources := mc.Interface.ListEgressPoliciesForSourceIdentity(serviceIdentity.ToK8sServiceAccount())
```

Comment:
may not be appropriate in this PR but it would be clear if it was `mc.ComputeInterface`

Reply:
Agree but it's better to be addressed in a separate PR.

Reply:
the issue is the stutter of `compute.ComputeInterface`. 

Whatever we decide, this is an embedded interface, so we can simply call `mc.ListEgressPoliciesForSourceIdentity`. This is useful, because a caller outside of the meshCatalog package doesn't really need to know which one has the responsbility